gnd/src/test/java/com/google/android/gnd/persistence/local/LocalDataStoreTest.java

```diff
@@ -569,4 +678,37 @@ public void testGetOfflineAreas() {
         .test()
         .assertValue(ImmutableList.of(TEST_OFFLINE_AREA));
   }
+
+  @Test
+  public void testStringToList() {
+    assertThat(FeatureEntity.stringToList(POLYGON_VERTICES_STRING))
+        .isEqualTo(POLYGON_VERTICES_LIST);
+  }
+
+  @Test
+  public void testStringToList_emptyValue() {
+    assertThat(FeatureEntity.stringToList("")).isEqualTo(null);
+  }
+
+  @Test
+  public void testStringToList_nullValue() {
+    assertThat(FeatureEntity.stringToList(null)).isEqualTo(null);
+  }
+
+  @Test
+  public void testListToString() {
+    assertThat(FeatureEntity.listToString(POLYGON_VERTICES_LIST))
+        .isEqualTo(POLYGON_VERTICES_STRING);
+  }
+
+  @Test
+  public void testListToString_emptyList() {
+    assertThat(FeatureEntity.listToString(ImmutableList.<Point>builder().build()))
```

Comment:
You can use `ImmutableList.of()` instead to create an empty list.

Reply:
replaced